Backend/Utils/MySqlContext.cs

```diff
@@ -95,15 +95,202 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
-            modelBuilder.Entity<Period>().HasNoKey();
-            modelBuilder.Entity<Occupation>().HasNoKey();
-            modelBuilder.Entity<Shift>().HasNoKey();
+            modelBuilder.Entity<City>().HasData(
+
+                 new City { Id = 21000, Name = "Novi Sad", StateId = 1 },
+                 new City { Id = 11000, Name = "Beograd", StateId = 1 }
+                 );
+
             modelBuilder.Entity<State>().HasData(
-                new State { Id=1, Name="Serbia"}
+                new State { Id = 1, Name = "Serbia" }
+            );
+
+            modelBuilder.Entity<Address>().HasData(
+                new Address { Id = 1, Street = "Radnicka", Number = 4, CityId = 21000 },
+                new Address { Id = 2, Street = "Gospodara Vucica", Number = 5, CityId = 11000 }
                 );
-            modelBuilder.Entity<City>().HasData(
-                new City { Id=21000, Name="Novi Sad", StateId=1}
+
+            modelBuilder.Entity<InsurancePolicy>().HasData(
+                new InsurancePolicy { Company = "Dunav osiguranje d.o.o", Id = "policy1", StartTime = new DateTime(2020, 11, 1), EndTime =  new DateTime(2020, 11, 1) }
+            );
+
+            modelBuilder.Entity<RegisteredUser>().HasData(
+                new RegisteredUser
+                {
+                    Id = "2406978890045",
+                    CurrResidenceId = 1,
+                    DateOfBirth = new DateTime(1978, 6, 24),
+                    DateOfCreation = new DateTime(),
+                    EducationLevel = EducationLevel.bachelor,
+                    Email = "marko@gmail.com",
+                    Gender = Gender.MALE,
+                    InsurancePolicyId = "policy1",
+                    Name = "Marko",
+                    Surname = "Markovic",
+                    Username = "markic",
+                    Password = "marko1978",
+                    Phone = "065/123-4554",
+                    PlaceOfBirthId = 11000,
+                    Profession = "vodoinstalater",
+                    ProfileImage = "."
+
+                }
                 );
+
+            modelBuilder.Entity<Feedback>().HasData(
+                new Feedback { Id = 1, AdditionalNotes = "Sve je super!", Approved = true, Date = new DateTime(), RegisteredUserId = "2406978890045", Anonymous = false, AllowedForPublishing = true },
+                new Feedback { Id = 2, AdditionalNotes = "Bolnica je veoma losa, bas sam razocaran! Rupe u zidovima, voda curi na sve strane, treba vas zatvoriti!!!", Approved = false, Date = new DateTime(), RegisteredUserId = "2406978890045", Anonymous = false, AllowedForPublishing = true },

```

Comment:
same applies here

Reply:
Formatted all entities that were hard to read